server/models/ParticipantsModel/index.js

```diff
@@ -106,17 +106,13 @@ const allParticipantsQuery = (user, queryParams) => {
           $cond: {
             if: { $ne: [$synced, null] },
             then: {
-              $dateDiff: {
-                startDate: { $toDate: $Consent },
-                endDate: { $toDate: $synced },
-                unit: timeUnit,
+              $floor: {
+                $divide: [{ $subtract: [$synced, $Consent] }, 86400000],
```

Comment:
can we extract the "magic number" `86400000` to a constant?

Reply:
[b744a93](https://github.com/AMP-SCZ/dpdash/pull/652/commits/b744a9333a191c2646787c9ad8663ba2b9455ea3)